packages/python/patch-no-srand

```diff
@@ -0,0 +1,23 @@
+--- Modules/expat/xmlparse.c
++++ Modules/expat/xmlparse.c
+@@ -20,7 +20,9 @@
+ #include <string.h>                     /* memset(), memcpy() */
+ #include <assert.h>
+ #include <limits.h>                     /* UINT_MAX */
++#if !defined(__CloudABI__)
```

Comment:
It has the advantage that it reduces the total size of the diff, meaning it's less likely to break when updating (e.g., when upstream adds additional includes).